x-pack/plugins/lists/common/schemas/request/create_list_schema.test.ts

```diff
@@ -9,23 +9,47 @@ import { pipe } from 'fp-ts/lib/pipeable';
 
 import { exactCheck, foldLeftRight, getPaths } from '../../siem_common_deps';
 
-import { getListRequest } from './mocks/utils';
-import { createListSchema } from './create_list_schema';
+import { CreateListSchema, createListSchema } from './create_list_schema';
+import { getCreateListSchemaMock } from './create_list_schema.mock';
 
 describe('create_list_schema', () => {
-  // TODO: Finish the tests for this
   test('it should validate a typical lists request', () => {
-    const payload = getListRequest();
+    const payload = getCreateListSchemaMock();
```

Comment:
I am being careful to abstract tests too much or add utils that reduce too much repetition as that could cause later maintainers to have a harder time with fixing a single util that suddenly has 5+ tests breaking all at once.

I might later but I usually just repeat things within tests where needed and then later if I find myself constantly maintaining a chunk I might make a utils. Usually though, once you write a batch of tests they stick around for a very very long time with little maintenance other than additive changes or small changes during refactoring that make you "feel better" because you see that you're changing all the tests in the same fashion through each of the end points you change.

The line we kind of tread on of "pick your poison" would be repetitive copy-pasta of tests vs. inventing test utils and test harnesses that others cannot easily figure out or later have to un-do because although the tests at one point were able to share the same utils/code they suddenly cannot and the maintainer has to begin undoing the utils.